### PR TITLE
[IMP] point_of_sale,pos_*: new receipt screen

### PR DESCRIPTION
**Pull request by caburj**
Aside from new look, we also move the email option from payment
screen to receipt screen. This allows emailing the receipt to a
customer that is not in the contact list.

We also removed invoicing in the receipt screen during failed invoicing
from payment screen as it is removed in here: bc48e00

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/static/src/xml/Screens/ReceiptScreen/ReceiptScreen.xml [hunk 1]
```diff
@@ -6,27 +6,53 @@
...
+                            </div>
+                        </form>
+                        <t t-if="orderUiState.emailFailed">
+                            <div class="notice failed">
+                                <t t-esc="orderUiState.emailFailedMessage"></t>
+                            </div>
+                        </t>
+                    </div>
+                </div>
+                <div class="done-view" t-else="">
```
Comment (pimodoo) [inline addons/point_of_sale/static/src/xml/Screens/ReceiptScreen/ReceiptScreen.xml:46]: @caburj I am not sure we need this done view, it create one more step before going to the next order. Do you think we could remove it? as we already get errors if something went wrong.

Did you try this nex screen with the IOT box, and the option automatically printing, to be sure that erverything is oK?

Thanks

Comment (caburj) [reply addons/point_of_sale/static/src/xml/Screens/ReceiptScreen/ReceiptScreen.xml:46]: It indeed add one more step. It is in the mockup so I added it. I haven't tried printing with iot box.
- I'll remove the done-view as I agree with it.
- I'll test the printing with IoT box.

Comment (pimodoo) [reply addons/point_of_sale/static/src/xml/Screens/ReceiptScreen/ReceiptScreen.xml:46]: cool, otherwise, ti seems really good

**Comment by pimodoo** [comment]
@caburj Also, I've just seen there is a conflict in the css file, probably, because I've merged the Robin's task.

**Comment by caburj** [comment]
@pimodoo I made the requested check and changes. You can proceed in the review. :)

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**